pyanaconda/geoloc.py

```diff
@@ -338,12 +334,13 @@ def __init__(self,
                             once the class is initialized
         :type refresh_now: bool
         """
-        self._provider = _get_provider(provider_id)
+        provider = AVAILABLE_GEOLOCATION_PROVIDERS.get(provider_id, FedoraGeoIPProvider)
```

Comment:
Yeah, we really don't need this as a constant - so I've moved it to a local variable.